python/filtering/filter_menu.py

```diff
@@ -201,20 +205,21 @@ def __init__(
             self.__undock_widget.setCheckable(False)
             self.__undock_widget.setIcon(SGQIcon.red_bullet())
             self.__undock_widget.clicked.connect(self.undock_filters)
-            # Clear All Filtesr button for dock widget
+            # Clear All Filters button for dock widget
             self.__clear_widget = SGQToolButton(self.dock_widget)
             self.__clear_widget.setObjectName(
                 "filter_menu_dock_widget_clear_all_filters_button"
             )
             self.__clear_widget.setChecked(False)
             self.__clear_widget.setCheckable(False)
-            self.__clear_widget.setText("Clear All Filters")
+            self.__clear_widget.setText("Clear Filters")
             sizePolicy = QtGui.QSizePolicy(
                 QtGui.QSizePolicy.Maximum, QtGui.QSizePolicy.Maximum
             )
             self.__clear_widget.setSizePolicy(sizePolicy)
+            # TODO should it be clearing preset filters here?
             self.__clear_widget.clicked.connect(self.clear_filters)
```

Comment:
I've not seen an example of this bit of code being hit. I suspect I should update the connection to use a lambda that calls the clear_filters with the clear_preset_filters

Reply:
This clear widget is used when the filters are docked, and its behavior should be the same as the "Clear All Filters" menu action in the main Filter menu

![image](https://github.com/user-attachments/assets/96b604d5-85d2-408b-b208-a9e3ba5ba810)

Reply:
yeah the issue I have here is that the named filters aren't implemented for the breakdown app so I had no way to test the changes here. I agree that it would need to be there if it was used. Usually in the past we've not implemented bits if they are not used.
Maybe I can add a test around it, even if nothing actually uses the functionality in practice (at least for the time being).